src/themes/money/theme.json

```diff
@@ -1369,7 +1369,8 @@
           "borderRadius": "50%",
           "borderStyle": "solid",
           "borderWidth": "2px",
-          "content": "' '",
+          "content": "'....'",
```

Comment:
would a width not work?

Reply:
Width and height are currently set to `17px`

Reply:
Setting `"flex": "none"` instead is a much nicer solution, thanks @jonathan-fielding